packages/dd-trace/test/instrumenter.spec.js

```diff
@@ -352,4 +352,27 @@ describe('Instrumenter', () => {
       })
     })
   })
+
+  describe('with plugins disabled via configuration option', () => {
```

Comment:
A test case is missing for the interaction with `tracer.use()`.

Reply:
updated